packages/core/integration-tests/test/javascript.js

```diff
@@ -586,30 +584,26 @@ describe('javascript', function() {
     assert.equal(output(), 3);
   });
 
-  it.skip('should support importing a URL to a raw asset', async function() {
+  it('should support importing a URL to a raw asset', async function() {
     let b = await bundle(
       path.join(__dirname, '/integration/import-raw/index.js')
     );
 
-    await assertBundles(b, {
-      name: 'index.js',
-      assets: ['index.js', 'test.txt'],
-      childBundles: [
-        {
-          type: 'map'
-        },
-        {
-          type: 'txt',
-          assets: ['test.txt'],
-          childBundles: []
-        }
-      ]
-    });
+    await assertBundles(b, [
+      {
+        name: 'index.js',
+        assets: ['index.js', 'test.txt.js']
+      },
+      {
+        type: 'txt',
+        assets: ['test.txt']
+      }
+    ]);
 
     let output = await run(b);
     assert.equal(typeof output, 'function');
-    assert(/^\/test\.[0-9a-f]+\.txt$/.test(output()));
-    assert(await fs.exists(path.join(__dirname, 'dist/', output())));
+    assert(/^\/dist\/test\.[0-9a-f]+\.txt$/.test(output()));
```

Comment:
we should probably support `outDir` as a parcel option again (and default it to "dist"), and not hardcode these things into the namers. When we write in these relative urls, we don't know any prefix the namer has applied. I imagine in most cases that prefix will not be part of the `publicUrl`.